conf.yaml

```diff
@@ -750,6 +751,17 @@ contents:
                 repo:   cloud
                 path:   docs/shared
                 exclude_branches: [ 1.0 ]
+              -
+                repo:   cloud-assets
+                path:   docs
+                map_branches: &mapCloudEceToCloudAssets
```

Comment:
You only need to add the ` &mapCloudEceToCloudAssets` if you will reuse the source.